docs/source/testing/integration.rst

```diff
@@ -18,4 +18,135 @@
 Writing Integration Tests for NWChemEx
 ######################################
 
-TODO: Write me!!!!
+NWChemEx is a modular ecosystem designed with separation of concerns as a key
+design point. An example of this separation can be found with the 
+SCF, integrals, ChemCache libraries. These components of NWX are linked by 
+SimDE and are intended to be used together, but are not explicitly required for 
+the development of the other (see :numref:`fig_deps_chart`). The unit tests for 
+these libraries are intended to ensure basic functionality and correctness, 
+which can usually be accomplished with simple test data that allow the unit 
+tests to run quickly. 
+
+.. _fig_deps_chart:
+
+.. figure:: assets/DependencyChart.png
+    :align: center
+    :scale: 50 %
+
+    A simplified diagram of the NWChemEx dependency structure. Note that Plugins
+    depending on SimDE are parallel to one another, and are integrated together
+    within NWChemEx. Arrows point from a dependency to the dependent library.
+
+
+With that said, the initial development and testing of the SCF becomes very 
+awkward when one is unable to easily acquire real integrals for real molecular 
+systems. Additionally, changes to the integrals code could have deleterious
+effects on the SCF code, which we would like to detect before merging. For these
+(and other reasons), it can be useful to implement integration tests to ensure
+the continued interoperability of the isolated components of the NWX stack.
+Because the tests are built on top of the Plugins, it is simple to include
```

Comment:
Plugins or plugins?

Reply:
I don't think we have a specific capitalization convention for "plugin" yet, but changing to lowercase is fine with me.